docs/scripts/preview.sh

```diff
@@ -20,7 +20,7 @@ cleanup()
 rm -rf $BUILD_DIR
 mkdir $BUILD_DIR
 
-bazel build //docs:docs
-tar -zxf ../../bazel-bin/docs/html.tar.gz -C $BUILD_DIR
+bazel build //docs:docs-no-pdf
```

Comment:
I'd like to be able to preview docs with PDF from time to time. Could we make this a separate script, or have a flag to set pdf/no-pdf ,please? Happy for no pdf to be the default.

Reply:
What does a script give you over `bazel build //docs:pdf-docs` and opening the resulting pdf? The preview script for the html stuff makes sense since there isn’t a single file that you can just open as a result, for the pdf I don’t see any value in keeping a script.